<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
    <meta name="viewport" content="width=320"/>
    <title>A speciation gene for left–right reversal in snails results in anti-predator adaptation</title>
    <link rel="stylesheet" type="text/css" href="../style.css"/>
  </head>
  <body class="articletype_article"><div class="category">Article</div><h1>A speciation gene for left–right reversal in snails results in anti-predator adaptation</h1><div class="authors">Masaki Hoso, Yuichi Kameda, Shu-Ping Wu, Takahiro Asami, Makoto Kato &amp; Michio Hori</div><div class="book"></div><div class="abstract"><p>How speciation genes can spread in a population is poorly understood. In land snails, a single gene for left–right reversal could be responsible for instant speciation, because dextral and sinistral snails have difficulty in mating. However, the traditional two-locus speciation model predicts that a mating disadvantage for the reversal should counteract this speciation. In this study, we show that specialized snake predation of the dextral majority drives prey speciation by reversal. Our experiments demonstrate that sinistral <i>Satsuma</i> snails (Stylommatophora: Camaenidae) survive predation by <i>Pareas iwasakii</i> (Colubroidea: Pareatidae). Worldwide biogeography reveals that stylommatophoran snail speciation by reversal has been accelerated in the range of pareatid snakes, especially in snails that gain stronger anti-snake defense and reproductive isolation from dextrals by sinistrality. Molecular phylogeny of <i>Satsuma</i> snails further provides intriguing evidence of repetitive speciation under snake predation. Our study demonstrates that a speciation gene can be fixed in populations by positive pleiotropic effects on survival.</p></div><div class="body"><div class="figures_preview"><b>Figures at a glance</b><div class="figure_preview"><a href="fig://f1"><img src="ncomms1133-f1.jpg"/></a></div><div class="figure_preview"><a href="fig://f2"><img src="ncomms1133-f2.jpg"/></a></div><div class="figure_preview"><a href="fig://f3"><img src="ncomms1133-f3.jpg"/></a></div><div class="figure_preview"><a href="fig://f4"><img src="ncomms1133-f4.jpg"/></a></div><div class="figure_preview"><a href="fig://f5"><img src="ncomms1133-f5.jpg"/></a></div></div>
<p>Speciation occurs when populations accumulate genetic differences that, upon secondary contact, cause reproductive incompatibily  <a href="ref://b1">1</a>. How genes for reproductive incompatibilities (speciation genes) could spread in a population however, has been only poorly understood  <a href="ref://b2">2</a> <a href="ref://b3">3</a> <a href="ref://b4">4</a>. In land snails, instant speciation may occur once a population becomes fixed for left–right reversal by a single gene. Copulation between dextral (clockwise coiled) and sinistral (counterclockwise coiled) snails is usually strongly impeded by genital and behavioural mismatches, because they are reversed in genital position on the body side as well as in coiling direction  <a href="ref://b5">5</a> <a href="ref://b6">6</a> <a href="ref://b7">7</a>. The reversal is determined by a maternal effect of a single nuclear gene in at least four phylogenetically segregated families of pulmonate snails  <a href="ref://b6">6</a> <a href="ref://b8">8</a> <a href="ref://b9">9</a> <a href="ref://b10">10</a>. Thus, fixation for a reversal allele in an initially spatially isolated population could complete premating isolation between dextral and sinistral populations, giving rise to a new species  <a href="ref://b6">6</a> <a href="ref://b7">7</a> <a href="ref://b11">11</a> <a href="ref://b12">12</a> <a href="ref://b13">13</a>. In fact, left–right reversal has evolved many times, especially in terrestrial snails  <a href="ref://b14">14</a> <a href="ref://b15">15</a>.</p>
<p>However, positive frequency-dependent selection resulting from difficulties of mating with the majority coiling phenotype is expected to eliminate the derived phenotype with reversed chirality  <a href="ref://b5">5</a> <a href="ref://b16">16</a>. Thus, as the two-locus model  <a href="ref://b17">17</a> <a href="ref://b18">18</a> <a href="ref://b19">19</a> predicts, speciation caused by a reproductive incompatibility at a single locus should only rarely proceed, without either genetic drift  <a href="ref://b11">11</a> <a href="ref://b13">13</a> or natural selection for the reversal that more than offsets the mating disadvantage  <a href="ref://b12">12</a> <a href="ref://b20">20</a>. Therefore, the origins of reversed snails have been puzzling.</p>
<p>Southeast Asian snakes in the family Pareatidae are regarded as dietary specialists of terrestrial snails and slugs  <a href="ref://b21">21</a> (<a href="fig://f1">Fig. 1a</a>). Because of the overwhelming predominance of dextral snails  <a href="ref://b14">14</a>, most pareatid snakes have evolved outstanding asymmetry in mandibular tooth number (<a href="fig://f1">Fig. 1b</a>), which probably facilitates feeding on dextral snails  <a href="ref://b22">22</a>. The Japanese pareatid, <i>P iwasakii</i>, strikes an exposed soft part of a crawling snail from behind at a determined leftward angle to grasp the ventral shell surface with the upper jaw and the soft body by inserting the mandibles into the aperture. The snake then extracts the soft body by alternately protracting and retracting the asymmetric right and left mandibles (  <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Movie 1</a>). Because of this 'right handedness' of sequential behaviour, the snake can rarely grasp a sinistral snail (  <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Movie 2</a>). This survival advantage of sinistral variants may more than offset the mating disadvantage in dextral species and enhance the chance of population fixation for left–right reversal, which completes premating isolation  <a href="ref://b22">22</a>.</p>
<p>Our hypothesis predicts that sinistral lineages of snails more frequently emerge in sympatry with the snakes than in allopatry. Furthermore, the effect of snake predation on the emergence of sinistral lineages should be larger in groups of flat and/or large species for the following reasons. First, interchiral mating is almost impossible in flat snails which generally perform two-way face-to-face copulation, whereas it is relatively easy for tall snails which copulate by shell mounting  <a href="ref://b5">5</a> <a href="ref://b6">6</a> <a href="ref://b7">7</a> <a href="ref://b16">16</a>. Thus, populations are less likely to be fixed for the reversal solely by genetic drift in flat snails than in tall snails  <a href="ref://b5">5</a> <a href="ref://b13">13</a>. Second, snakes are generally gape limited  <a href="ref://b21">21</a>, and pareatid snakes cannot even hold a sinistral snail to prey upon unless the shell aperture is small enough  <a href="ref://b22">22</a> (  <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Movie 2</a>). Thus, the relative advantage of sinistrality under snake predation should also be larger in species with larger shells.</p>
<p>Here, we tested these predictions by worldwide analyses of distributions of dextral and sinistral stylommatophoran snails and pareatid snakes. Furthermore, we examined the possible role of snake predation in the evolution of multiple sinistral lineages by the analyses of molecular phylogeny and biogeography of snails in the genus <i>Satsuma</i>. This study illustrates how a single gene for reproductive incompatibility could generate a new species by natural selection for ecological adaptation.</p>
<div class="section seclevel1" id="10278569920997950395">
<h2>Results</h2>
<div class="section seclevel2" id="16418491178845717320">
<h2>Feeding experiments</h2>
<p>In our experiments with dextral and sinistral <i>Satsuma</i> species, none of the dextral snails survived snake predation (<a href="fig://f2">Fig. 2</a>;   <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Movie 1</a>). In strong contrast, 87.5% of the sinistral snails survived predation (  <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Movie 2</a>). The interaction effect of coiling direction and shell size on survival was significant (likelihood ratio test (LRT) for comparing generalized linear mixed models (GLMMs), d.f.=1, <i>χ</i><sup>2</sup>=10.8, <i>P</i>&lt;0.01; <a href="fig://f2">Fig. 2</a>) when the effects of snake individual and snail groups were statistically removed in the combined data set including the results of the study by Hoso <i>et al</i>.  <a href="ref://b22">22</a> This analysis, however, did not exclude the effects of interspecific differences in characteristics other than coiling direction, which potentially influence the survival rate. Throughout the experiment, snakes did not change the right-handed attacking behaviour. Snail extraction has commonly been observed in six other pareatid species (<i>Aplopeltura boa</i>, <i>P. boulengeri</i>, <i>P. carinatus</i>, <i>P. formosensis</i>, <i>P. margaritophorus</i> and <i>P. stanleyi</i>).</p>
<p>This advantage of sinistrality results from a mechanical mismatch between the snake's right-handed predation behaviour and the snail's left-handed coiling structure. The snake does not modify the right-handed attacking behaviour. Thus, to hold a sinistral snail, the snake must reach the upper jaw onto the dorsal shell surface across the aperture, while inserting the lower mandibles under the soft body. However, due to the gape limitation  <a href="ref://b21">21</a>, the snake can hardly achieve it when the snail's aperture is relatively large. For this reason, the survival advantage of sinistrality was much larger in snails of the genus <i>Satsuma</i> compared with <i>Bradybaena similaris</i>. Thus, the result of our feeding experiment suggests that the survival advantage of sinistrality varies depending on the shell size, and thus the evolution of sinistral species in response to the right-handed predation by pareatid snakes may be more frequent in groups of larger snails.</p>
</div>
<div class="section seclevel2" id="16418730990798572089">
<h2>Comparative survey</h2>
<p>We found that sinistral genera are marginally associated with snail-eating pareatid snakes in geographical distribution (<a href="table://t1">Table 1</a>; LRT for comparing GLMMs, d.f.=1, <i>χ</i><sup>2</sup>=5.5, <i>P</i>=0.019). The statistical test incorporating distribution, shape and their interaction indicated that sinistral genera are more strongly associated with the sympatry with the snakes in flat snails than that in tall snails (LRT for comparing GLMMs, d.f.=1, <i>χ</i><sup>2</sup>=9.8, <i>P</i>&lt;0.01; <a href="fig://f3">Fig. 3a,b</a>). The effect of interaction between distribution and size was also significant (LRT for comparing GLMMs, d.f.=1, <i>χ</i><sup>2</sup>=8.6, <i>P</i>&lt;0.01; <a href="fig://f3">Fig. 3c</a>). Thus, the association of sinistral genera with the snakes is stronger in flat snails than in tall snails and larger snails. These findings cannot be explained by a phylogenetic artifact because the effect of higher taxa was statistically removed in every analysis by incorporating higher taxa (family) as a random effect (<a href="fig://f4">Fig. 4</a>).</p>
<p>In a LRT between generalized linear models (GLMs) that accounts for shell sizes of dextral and sinistral congeners (33 genera), the interaction of the effect of snakes and coiling direction on shell size was not significant (LRT for comparing GLMs, <i>χ</i><sup>2</sup>=0.126, d.f.=1, <i>P</i>=0.723). This suggests that snake predation is not likely to be responsible for the shell size of sinistral snails.</p>
</div>
<div class="section seclevel2" id="16418970802751426858">
<h2>Phylogenetic analyses</h2>
<p>In our phylogenetic analyses of the snail genus <i>Satsuma</i> in Japan and Taiwan (<a href="fig://f5">Fig. 5a</a>), the combined data matrix consisted of 2,842 nucleotide sites, of which 779 were parsimony informative. The maximum parsimony (MP) analysis resulted in the two most parsimonious trees (5,337 steps, consistency index excluding uninformative characters=0.2631, retention index=0.6068). The result of maximum likelihood (ML) and Bayesian analyses were largely consistent with the result of the MP analysis (<a href="fig://f5">Fig. 5b</a>,   <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Fig. S1</a>). In both the combined and separate data sets, the likelihood values of the constrained trees for sinistral lineage to be monophyletic were significantly lower than those of the unconstrained best trees (the approximately unbiased (AU) test  <a href="ref://b23">23</a> <a href="ref://b24">24</a>, all <i>P</i>-values &lt;0.01), indicating that the sinistral species have evolved from dextral ancestors multiple times within <i>Satsuma</i>.</p>
</div>
</div>
<div class="section seclevel1" id="10278809732950805164">
<h2>Discussion</h2>
<p>The results of feeding experiments support our hypothesis that under 'right-handed' predation by pareatid snakes, the survival advantage of sinistral variants may counterbalance their mating disadvantage. In this case, selective predation may facilitate population fixation for the sinistral allele and simultaneously cause speciation. This hypothesis predicts that the diversity of sinistral taxa is higher in the range of the snail-eating snakes.</p>
<p>Our biogeographical analyses supported this prediction. Pareatid snakes may have continued specialized predation since their early phylogenetic segregation in the Colubroidea  <a href="ref://b22">22</a> <a href="ref://b25">25</a>. No terrestrial snail predators other than the snakes are known to specialize for prey handedness  <a href="ref://b26">26</a>. Therefore, our results are consistent with the hypothesis that 'right-handed' snake predation drives snail speciation by left–right reversal. Further experimental studies may be desired to examine the effect of shell shape on the snake's predation success  <a href="ref://b27">27</a>.</p>
<p>The analysis of phylogeny was necessary to distinguish speciation by reversal from monophyletic diversification of sinistrals. However, no phylogenetic hypothesis of snail taxa in a global scale has been available. As a test case, we examined the molecular phylogeny of the genus <i>Satsuma</i>, which includes multiple dextral and sinistral species in the snake range. In natural conditions, <i>P. iwasakii</i> preys on dextral species of <i>Satsuma</i> <a href="ref://b28">28</a>, which exhibit defensive aperture morphology against snake predation  <a href="ref://b27">27</a>. Thus, <i>Satsuma</i> snails have probably been subject to severe snake predation. Superiority of sinistral <i>Satsuma</i> species in our feeding experiments suggests that the phylogeny of <i>Satsuma</i> may reflect the history of prey–predator interactions with the snakes.</p>
<p>The presence of multiple sinistral <i>Satsuma</i> lineages within the monophyletic genus can hardly be explained by introgression via hybridization with dextral snails after completion of speciation, because <i>Satsuma</i> snails perform face-to-face copulation that prevents interchiral mating  <a href="ref://b5">5</a>. Thus, these sinistrals must have evolved in one of three possible scenarios: independent fixation for sinistrality in each lineage, a single event of fixation for a sinistrality that temporally represents ancestral polymorphism or combination of these.</p>
<p>Fixation for sinistrality should be rapid because both directional selection by snake predation and positive frequency-dependent selection cooperate for sinistrality once sinistral variants become more common than dextrals under the snake predation, while other populations of the ancestral species remain dextral. Consequently, a sinistral species could be composed of non-sibling populations unless lineage sorting is complete, as seen in the polyphyletic mitochondrial lineages of a dextral species derived from a sinistral <i>Euhadra</i> species outside the range of pareatid snakes  <a href="ref://b20">20</a>. In contrast, our multilocus analyses strongly supported the independent origins of sinistral <i>Satsuma</i> by deep branching into three clades.</p>
<p>Overall, our results suggest that the mating disadvantage of left–right reversal has independently been resolved in the multiple sinistral lineages. These events of evolution to sinistrals may be too frequent in such a narrow geographical range to attribute to fixations solely by genetic drift. The virtual absence of sinistral <i>Satsuma</i> outside the range of pareatid snakes supports the likelihood that left–right reversal is higher under predation by the snakes. Relatively recent repetition of emergence of a sinistral lineage also suggests that right-handed predation by the snakes may also currently be creating new sinistral species by left–right reversal in <i>Satsuma</i>.</p>
<p>However, our result does not rule out the possibility that the proportion of sinistral taxa per lineage might have apparently been amplified by extinctions of dextral taxa because of snake predation. The presence of sinistral genera outside the snake range indicates that sinistral species have also evolved without snake predation, for example, resulting from reproductive characteristic displacement  <a href="ref://b20">20</a> <a href="ref://b29">29</a> <a href="ref://b30">30</a>, or stochastic fluctuation  <a href="ref://b6">6</a>. These events should be as frequent in sympatry with the snail-eating snakes as in allopatry. Thus, we do not conclude that snake predation is responsible for the origin of every sinistral snail species inside the snake ranges.</p>
<p>One primary prey for a snake predator disappears through each event of population fixation for reversal. Local accumulation of sinistral snails may eventually diminish the advantage of right handedness for the snake and prompt the counteradaptive evolution of 'left-handed' snakes, just as the handedness ratio oscillates in scale-eating cichlid fishes  <a href="ref://b31">31</a>. However, pareatid snakes with reversed dentition have never been found across Southeast Asia  <a href="ref://b22">22</a>. Yet, dextral snails remain the majority at genus level and predominant in the snake range. Indeed, dextral snails available for <i>P. iwasakii</i> are not only species-rich but also still abundant in snail communities  <a href="ref://b28">28</a>. Thus, the snakes could have shifted prey to less-protected alternatives and enjoyed the advantage of right handedness, as theories of optimal foraging and coevolutionary alternations  <a href="ref://b32">32</a> <a href="ref://b33">33</a> predict. These ecological differences between predator snakes and prey snails may underlie the absence of counteradaptive responses by pareatid snakes, allowing the prey to repeat evolutionary escape by speciation.</p>
<p>Generally, identification of an isolation barrier that initiated speciation is rarely possible because subsequent genetic and ecological divergence builds up additional barriers that confound analysis  <a href="ref://b34">34</a> <a href="ref://b35">35</a> <a href="ref://b36">36</a> <a href="ref://b37">37</a>. However, instant speciation of snails by left–right reversal is exceptional  <a href="ref://b34">34</a>, where the speciation gene may hardly change ecological features, such as resource use or habitat preference, except for susceptibility to snake predation and partners to mate. In the secondary contact with a new sinistral population (species), dextral ancestors without additional ecological divergence would go extinct because of a severe survival disadvantage under snake predation. Thus, not only reversal but also other mechanisms are required for the coexistence of dextral and sinistral congeners in <i>Satsuma</i>. Their coexistence may offer rare opportunities to distinguish post-speciation divergences from the cause of speciation, bridging our understandings of speciation and community assembly.</p>
<p>Our study demonstrated how positive selection could operate for major pleiotropic effects of a speciation gene on anti-predator survival, exemplifying a 'magic trait'  <a href="ref://b38">38</a> that simultaneously gives rise to adaptation and reproductive isolation. Speciation by natural selection may initially require genetically simpler changes of phenotype than previously expected. Our evidence for the evolutionary role of single-gene pleiotropy resurrects Goldschmidt's controversial idea of hopeful monsters  <a href="ref://b39">39</a>.</p>
</div>
<div class="section seclevel1" id="10279049544903659933">
<h2>Methods</h2>
<div class="section seclevel2" id="15967475677933462294">
<h2>Feeding experiments</h2>
<p>We conducted predation experiments between August 2004 and August 2005 at Kyoto University in accordance with the Institutional Regulation on Animal Experimentation. As predator snakes, we used four individuals of <i>P. iwasakii</i> (three from Ishigaki Island and one from Iriomote Island; snout-vent length: 430–580 mm; head width: 7.6–9.5 mm; two of the snakes were also used in the study by Hoso <i>et al</i>.  <a href="ref://b22">22</a>). As prey snails, we used snails from the genus <i>Satsuma</i>: <i>S. mercatoria</i> (dextral) in Okinawa Island and <i>S. perversa</i> (sinistral) in Kume Island, the Ryukyu Archipelago, Japan. <i>Satsuma</i> snails (mean±s.d.=27.4±1.9 mm in shell diameter) were larger than <i>B. similaris</i> (12.7±1.5 mm) used in the study by Hoso <i>et al</i>.  <a href="ref://b22">22</a> <i>S. mercatoria</i> and <i>S. perversa</i> are suitable for this study, because they resemble each other in shape and size and are closely related (<a href="fig://f5">Fig. 5b</a>). To exclude the potential effects of local adaptation and learning by predators and prey, we collected these snails from islands where no pareatid snakes occur.</p>
<p>To test the interactive effect of sinistrality and size of snails on survival rate from snake predation, we performed LRTs between null models and test models where the focal factor was additionally incorporated as a fixed effect. As covariates, we adopted coiling direction, shell diameter and snail groups (<i>Satsuma</i> spp. or <i>B. similaris</i>). We incorporated snake individuals as random effects to avoid pseudo-replication resulting from using the same snakes repeatedly.</p>
</div>
<div class="section seclevel2" id="15967715489886317063">
<h2>Comparative survey</h2>
<p>We classified all of the recent terrestrial snail genera of the order Stylommatophora by size, shape, coiling direction and distribution according to the information available in the literature (  <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Information</a>). We excluded taxa unsuitable for the analysis, that is, slugs with no or reduced shells, species that are chirally dimorphic in 11 genera  <a href="ref://b5">5</a> and the sinistral superfamily Clausilioidea, for the following reasons. Slug and chirally dimorphic taxa do not fit binary classification by coiling direction. Inclusion of the latter did not affect the result. The Clausilioidea is a cosmopolitan monophyletic taxon with a few exceptional dextral lineages, which is most likely to descend from the single sinistral ancestor. Thus, they do not provide meaningful values to test the distribution of reversal events from the dextral to the sinistral. Although it might be of use for the examination of opposite reversal from the sinistral to the dextral, it is irrelevant to the purpose of the present study. In this paper, we call a genus including sinistral species as a sinistral genus. Sympatric genera means that they are partly or fully sympatric with snail-eating pareatid snakes. In binary classification of genera, we treated 33 genera that include both dextral and sinistral species as the indication of separate reversal events. As the shell size of each genus, we used the shell diameter of the largest species in the genus because it was the only size indicator listed in every literature we referred. Shell diameter was log-transformed for the following parametric analysis.</p>
<p>Bimodal shell shapes are associated with discrete mating behaviours; tall-shelled snails usually mate by shell mounting, and globular- to flat-shelled snails mate face-to-face  <a href="ref://b5">5</a> <a href="ref://b40">40</a> albeit with several exceptions  <a href="ref://b41">41</a>. Probably because interchiral mating is physically easier and thus positive frequency-dependent selection is relaxed in species with shell-mounting copulation, sinistral genera have more frequently evolved in tall-shell shaped taxa in pulmonates  <a href="ref://b5">5</a> <a href="ref://b7">7</a>. Therefore, the shell shape may generate potentially important covariance in the test of our hypothesis. Tall genera were defined as having the shell height greater than the shell diameter, globular to flat genera as having the height equal to or less than the diameter. This simple definition reflects the bimodality in the shell shape of terrestrial stylommatophoran pulmonates, as originally reported by Cain  <a href="ref://b42">42</a> and subsequently by others (for example, ref.   <a href="ref://b43">43</a>). Genera were assigned to two degrees at ordinal scales; 342 tall genera into 1, and 485 flat genera and 11 genera containing both tall and flat species into 0. The examined data matrix is provided in   <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Data 1</a>.</p>
<p>To test the effects of snakes (sympatry or allopatry with pareatid snakes; test 1), interaction between snakes and size (maximum shell-diameter; test 2) and interaction between snakes and shape (tall or flat-shell shape), on the occurrence of sinistral genera (test 3), we separately performed LRTs between null models and test models where each focal factor was additionally incorporated as a fixed effect. As covariates, we adopted size and shape for test 1, and snakes, size and shape for test 2 and test 3. We incorporated the family to which each snail genus belongs as a random factor to avoid the potential bias due to the increase of sinistral genera by monophyletic radiation. Because multiple families include sinistral genera regardless of shell shape (<a href="fig://f4">Fig. 4</a>), local distributions of particular snail taxa did not affect our global scale analysis. Additionally, we tested an alternative hypothesis that snake predation favours the evolution of larger shells in sinistral snails instead of that of sinistrality in larger dextral snails.</p>
<p>We analysed the data using R version 2.11.1 in Windows  <a href="ref://b44">44</a> with the lme4 and glm packages. The summaries of analyses are provided in   <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Table S1</a>.</p>
</div>
<div class="section seclevel2" id="15967955301839171832">
<h2>Molecular methods</h2>
<p>To reconstruct a species level phylogeny, we sampled 75 species of <i>Satsuma</i>, including all species that inhabit the Ryukyu Archipelago except a sinistral species (<i>S. tadai</i>) endemic to Uotsuri Island (  <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Data 2</a>). We sequenced partial regions of mitochondrial cytochrome <i>b</i> oxidase subunit 1 gene (<i>CO1</i>) and nuclear ribosomal external transcribed spacer (<i>ETS</i>) and complete sequence of nuclear ribosomal internal transcribed spacers (<i>ITS</i>) 1 and 2. DNA extraction, PCR amplification, and sequencing of <i>CO1</i> and <i>ITS</i> were performed following the previously described method  <a href="ref://b45">45</a>. PCR amplification of <i>ETS</i> fragment was performed using primers Satsuma-ETSf (5′-CTCCAGCCCGGCGTWTGGCG-3′) and 18Sop (5′-GCTCCCTCTCCGGAATCGAACCC-3′) under the following conditions: initial denaturation for 3 min at 94°C, followed by 30 cycles of 30 s at 94°C and 1 min at 72°C, with a final extension for 3 min at 72°C. Sequencing of ETS region was performed using primers Satsuma-ETSf and Satsuma-ETSr (5′-TTTAATGAGCCATTCGCGG-3′). The obtained sequence data have been uploaded to GenBank and the accession codes are provided in   <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Data 2</a>.</p>
</div>
<div class="section seclevel2" id="15968195113792026601">
<h2>Phylogenetic analyses</h2>
<p>Alignment of the CO1 sequences was straightforward with no gaps. ITS and ETS sequences were aligned using ClustalX  <a href="ref://b46">46</a> with manual adjustments. Gaps were treated as missing data. For outgroups, we used three camaenid species, <i>Nipponochloritis hiromitadae</i>, <i>N. oscitans iwatensis</i> and <i>Yakuchloritis albolabris</i>. We conducted phylogenetic analyses of the combined data of three genes based on the MP, ML and Bayesian methods. MP analyses were performed using PAUP* 4.0b10  <a href="ref://b47">47</a>. We performed heuristic searches with ten random additional analyses and tree bisection-reconnection (TBR) branch swapping. The resulted trees were used as starting trees in an additional round of heuristic search with TBR branch swapping. Bootstrap analysis was performed based on 1,000 replicates of heuristic searches with ten random addition analyses and TBR branch swapping, saving no more than 100 trees per analysis. Before the ML and Bayesian analyses, we determined an appropriate model of sequence evolution and model parameters using Kakusan version 3.0  <a href="ref://b48">48</a>. ML analyses were performed using Treefinder  <a href="ref://b49">49</a> with the 100 iterations of the likelihood ratchet implemented in Phylogears  <a href="ref://b50">50</a>. Bootstrap analysis was conducted based on 1,000 replications. Bayesian analyses were performed using MrBayes v3.1.2  <a href="ref://b51">51</a>. The analysis was carried out by two simultaneous runs, each of which consisted of running four simultaneous chains for six million generations and sampling trees every 1,000 generations. We discarded first 1,501 trees as burn-in, and the remaining 9,000 samples were used to estimate the tree topology, branch length, and substitution parameters.</p>
<p>We tested the hypothesis of the monophyly of the sinistral <i>Satsuma</i> lineages by the AU test  <a href="ref://b23">23</a> <a href="ref://b24">24</a> for both combined and separate data sets of the three genes. The alternative trees were obtained by ML heuristic search as described above under the topological constraint (sinistral lineage monophyletic), and the AU test was conducted based on 1,000,000 replications using Treefinder  <a href="ref://b49">49</a>. A phylogenetically informed test for the independence of two characters  <a href="ref://b52">52</a> cannot be adequately performed in this case, because the absence of the snakes does not favor the evolution of dextral snails by left–right reversal, and their presence does not always result in the evolution of sinistral snails.</p>
<p>Our hypothesis predicts the different rates of character transition in snail coiling between clades depending on whether they have been under snake predation or not. In this case, maximum likelihood estimation of ancestral traits is not reliable, because it assumes the equal character-transition rates across all the clades. To avoid this artefact, we estimated ancestral states based on the most parsimonious scenarios. Although we independently reconstructed ancestral character states based on ML and Bayesian methods as implemented in BayesTraits  <a href="ref://b53">53</a> for a clade including 37 OTUs (operational taxonomic units; listed at the top of   <a href="http://www.nature.com/ncomms/journal/v1/n9/abs/ncomms1133.html#supplementary-information">Supplementary Data 2</a>) completely coexisting with the snakes, the probabilities of sinistral state in the best models converged around 0.5 at most nodes.</p>
<p>Reconstructing the ancestral state of a functional trait that is subjected to stringent natural selection requires a caution as follows. If the directional selection for sinistrality continues for a long time, the proportion of dextral lineages may decrease, and thus the phylogeny inferred for extant species may underestimate the number of reversals to sinistrality. Thus, our result may have underestimated the frequency of reversal in snails because of using a genus or monophyletic lineage as a unit to count. This shortcoming is inevitable unless the entire evolutionary history in question could be recovered  <a href="ref://b54">54</a>.</p>
</div>
</div>
<div class="section seclevel1" id="10279289356856514702">
<h2>Author contributions</h2>
<p>M. Hoso produced the original idea, conducted predation experiments and carried out literature survey; M.H., Y.K. and S.P.W. obtained and analysed the data for molecular phylogeny. All the authors contributed to production of the paper.</p>
</div>
<div class="section seclevel1" id="10279529168809369471">
<h2>Additional information</h2>
<p><b>How to cite this article:</b> Hoso, M. <i>et al</i>. A speciation gene for left-right reversal in snails results in anti-predator adaptation. <i>Nat. Commun.</i> 1:133 doi: 10.1038/ncomms1133 (2010).</p>
</div>
<h2>Acknowledgements</h2><p>We thank A. Mori, H. Ota and F. Sato for facilitating our behavioural experiments. We also thank M. Brockhurst, S. Chiba, R. Cogni, R.H. Cowie, A. Davison, B. Grant, C.C. Hwang, M. Kawata, K. Momose, J. Murray, and P. Nosil for helpful comments. This work was partly supported by grants-in-aid for JSPS fellowship (08J02138) to M.H., and for scientific research A (18207002) to M.K. and B (20405007) to T.A. and by the Global COE Programs 'Center for ecosystem management adapting to global change (J03)' and 'Formation of a strategic base for biodiversity and evolutionary research from genome to ecosystem (A06)' of the Ministry of Education, Culture, Sports, Science and Technology (MEXT) of Japan.</p></div><div class="socialdrm">Downloaded by r.page@bio.gla.ac.uk on 8th Dec 2010 from 86.1.200.144</div></body>
</html>